plugins/versionpress/src/Cli/vp-composer.php

```diff
@@ -0,0 +1,121 @@
+<?php
+// NOTE: VersionPress must be fully activated for these commands to be available
+
+// WORD-WRAPPING of the doc comments: 75 chars for option description, 90 chars for everything else,
+// see http://wp-cli.org/docs/commands-cookbook/#longdesc.
+// In this source file, wrap long desc at col 97 and option desc at col 84.
+
+namespace VersionPress\Cli;
+
+use VersionPress\ChangeInfos\PluginChangeInfo;
+use VersionPress\DI\VersionPressServices;
+use VersionPress\Git\Committer;
+use VersionPress\Utils\Process;
+use VersionPress\VersionPress;
+use WP_CLI;
+use WP_CLI_Command;
+
+/**
+ * VersionPress CLI commands for Composer scripts.
+ */
+class VPComposerCommand extends WP_CLI_Command
+{
+    /**
+     * Commits all changes made by Composer.
+     *
+     * @subcommand commit-composer-changes
+     */
+    public function commitComposerChanges($args, $assoc_args)
+    {
+        global $versionPressContainer;
+
+        if (!VersionPress::isActive()) {
+            WP_CLI::error('VersionPress is not active. Changes will be not committed.');
+        }
+
+        /** @var Committer $committer */
+        $committer = $versionPressContainer->resolve(VersionPressServices::COMMITTER);
+        $changes = $this->detectChanges();
+        $installedPackages = $changes['installed'];
+        $removedPackages = $changes['removed'];
+        $updatedPackages = $changes['updated'];
+
+        $changeInfos = array_merge(
```

Comment:
This is an outdated code and now it looks like [this](https://github.com/versionpress/versionpress/pull/1065/files#diff-9d8b13eddfd1eed01669006028cf0a0cR65). Are you sure you are reviewing the whole PR?